src/index.html

```diff
@@ -9,6 +9,13 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <section class="clock">
+      <div class="clock__container hands">
+        <div class="hands__seconds hand"></div>
+        <div class="hands__minutes hand"></div>
+        <div class="hands__hours hand"></div>
```

Comment:
why do you need `hand` selector? See you've not used it
```suggestion
        <div class="hands__seconds"></div>
        <div class="hands__minutes"></div>
        <div class="hands__hours"></div>
```